clients/cobol-lsp-vscode-extension/src/__tests__/services/copybook/CopybookMessageHandlerTest.spec.ts

```diff
@@ -15,80 +15,199 @@
 import { resolveCopybookHandler } from "../../../services/copybook/CopybookMessageHandler";
 import { CopybookURI } from "../../../services/copybook/CopybookURI";
 import { SettingsService } from "../../../services/Settings";
-import { searchCopybookInExtensionFolder } from "../../../services/util/FSUtils";
 import * as vscode from "vscode";
 import { Utils } from "../../../services/util/Utils";
+import { E4ECopybookService } from "../../../services/copybook/E4ECopybookService";
+import {
+  e4eMock,
+  e4eResponseMock,
+  e4eResponseMockDataSetPrior,
+} from "../../../__mocks__/getE4EMock.utility";
+import path = require("path");
+import { globSync } from "glob";
+import * as fs from "fs";
 
 vscode.workspace.getConfiguration = jest.fn().mockReturnValue({
   get: jest.fn().mockReturnValue("testProfile"),
 });
 
 Utils.getZoweExplorerAPI = jest.fn();
+jest.mock("glob");
+jest.mock("url", () => {
+  return {
+    URL: jest.fn().mockImplementation((x) => {
+      return { href: x };
+    }),
+  };
+});
+(globSync as any) = jest.fn().mockImplementation((x: any) => [x]);
+(fs.realpathSync.native as any) = jest.fn().mockImplementation((x: any) => x);
+(vscode.Uri.file as any) = jest.fn().mockImplementation((x: any) => {
+  return { fsPath: x };
+});
+
+vscode.window.createOutputChannel = jest.fn().mockReturnValue({
+  appendLine: jest.fn(),
+});
```

Comment:
what is the point of mocking this function? Isn't it enough to just create the outputChannel mock itself?